src/nginxconfig/i18n/en/templates/domain_sections/rails.js

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2020 DigitalOcean
```

Comment:
```suggestion
Copyright 2021 DigitalOcean
```
2021, please, as this is a new file